components/AmbulanceCard.jsx

```diff
@@ -18,6 +19,27 @@ const AmbulanceCard = ({
         <>
             <div className="w-full">
                 <div className="bg-white rounded-lg shadow dark:bg-gray-1200 dark:text-gray-300">
+                    <div className="w-full">
+                        <p
```

Comment:
<img width="1065" alt="Screenshot 2021-04-24 at 11 26 34 PM" src="https://user-images.githubusercontent.com/61158210/115968248-91fc5080-a554-11eb-8b65-a140a78e89f7.png">

Seems to be breaking...

Reply:
Fixed 📍